docs/faq.md

```diff
@@ -0,0 +1,53 @@
+---
+id: faq
+title: Frequently Asked Questions
+sidebar_label: FAQ - Questions
+---
+
+## What is the recommended single-spa setup?
+We recommend a setup that uses ES modules + [import maps](https://github.com/WICG/import-maps) + [dynamic import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Dynamic_Imports) (or [SystemJS](https://github.com/systemjs/systemjs) to polyfill these if you need better browser support).  This setup has several advantages:
+1. Common libraries are easy to manage, and are only downloaded once. You can also preload these for a small speed boost as well using the standard preload spec.
+2. Sharing code / functions / variables is as easy as an import statement, just like in a monothlithic setup
+3. Lazy loading applications is easy, which enables you to speed up initial load times
+4. Each application (AKA microservice, AKA ES module) can be independently developed and deployed. Teams are enabled to work at their own speed, experiment (within reason as defined by the organization), deploy, and QA on thier own schedules.
+5. Working on one area of an app doesn’t require you to download the entire frontend codebase — you just run your microservice locally and the other ones are automatically pulled from what’s on your dev/stage/prod environment
+
+## Is there a performance hit to doing microservices this way?
+When setup in the [recommended way](#what-is-the-recommended-single-spa-setup), your code performance and bundle size will be the
+
+## I don’t quite understand what single-spa does?
+Another way to think of single-spa is that it is just a top level router. When one route is active, it downloads and executes the code for that route.
+
+## How can I have only one version of (React, Vue, Angular, etc.) on the page?
+Using the recommended setup, you setup your import map to download that once. Then, tell each application to _not_ bundle that application code; instead, it will given to you at runtime in the browser. See webpack’s “externals” docs for how to do this.
+
+You do have the option of _not_ excluding those libraries (for example if you want to experiment with a newer version of a different library alltogether) but be aware of the effect that will have on user's bundle sizes and application speed.
+
+## What are import maps?
+[Import maps](https://github.com/WICG/import-maps) improve the developer experience of ES modules by allowing you to write something like `import React from "react"` instead of needing to use an absolute or relative URL for your import statement.
```

Comment:
Should probably also point out that it's a "future spec" that browsers are considering adopting.